app/assets/src/components/ResultsFolder.jsx

```diff
@@ -1,12 +1,123 @@
 import React from "react";
 import Divider from "./layout/Divider";
 
+class OutputFile extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this.file = props.file;
+  }
+
+  download(url) {
+    if (url) {
+      location.href = `${url}`;
+    }
+  }
+
+  render() {
+    return (
+      <tr
+        className={`${this.file.url ? "" : "disabled-"}file-link`}
```

Comment:
re className all new code should use CSS Modules and camelCase. See ui/controls/Tabs.jsx and tabs.scss

Reply:
I couldn't get the modules to work. Will bear in mind for new CSS in the future and seek clarification